File: Unidad 2-Estructuras de Control/Ejercicio2.c
//Autores: Programa realizado por el Equipo2_CodePain
//Version 1.2
//Este programa recibe dos numeros y nos dice cual es mayor, menor o si ambos son iguales.

#include <stdio.h>

 int main () {

     //Entrada: Declaración de variables y lectura de dos numeros
    int a = 0, b = 0;

    scanf("%d %d",&a,&b);

     //Proceso: Comparacion de numeros
     //Salida: impresion de quien es mayor, menor o si son iguales
    if (a > b) {
        printf("%d > %d",a,b);
    } else
    {
        if (b > a){
            printf("%d > %d",b,a);
        } else
        {
            printf("Son iguales");
        }   
    }
}

File: Unidad 2-Estructuras de Control/Equipo2_CodePain/Ejercicio2.py
# Este programa recibe dos números y nos dice cual es mayor, menor o si ambos son iguales.


#Inicio declaración de variables

aNum = 0 #Primer número a leer
bNum = 0 #Segundo número a leer 
iguales =  "Los números son iguales" #String que se imprime si ambos números son iguales 

#Fin declaración de variables


#Inicio de lectura de números 
#Entrada: cualquier número real

aNum = float(input()) #Leer primer número 
bNum = float(input()) #Leer segundo número


#Inicio condicionales para asignación e impresión de quien es mayor, menor o si son iguales 
#Salida: impresión de quien es mayor, menor o si son iguales


if aNum > bNum:
    print(str(aNum) + " > " + str(bNum)) #La primera salida es el número más grande y la segunda el más chico
elif (aNum < bNum):
    print(str(bNum) + " > " +  str(aNum)) #La primera salida es el número más grande y la segunda el más chico
else: 
    print(iguales)
